.github/workflows/is-accl-release-build.yml

```
@@ -0,0 +1,71 @@
+        run: |
+          VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+          echo "Project version: $VERSION"
+          echo "::set-output name=version::$VERSION"
+
+      - name: Build with Maven
+        run: mvn clean install
+
+      # Locate the ZIP produced by the IS accelerator distribution module
+      - name: Locate ZIP file
+        id: locate_zip
+        run: echo "::set-output name=zipfile::$(find ./distribution/is-accelerator/target -name '*.zip' | head -n 1)"
```

actionlint (1.7.12)

[error] workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)